recipes/r-climate4r.climdex/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set version = '0.2.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-climate4r.climdex
+  version: {{ version|replace("-", "_") }}
+
+source:
+  git_url: https://github.com/SantanderMetGroup/climate4R.climdex
+  git_tag: v0.2.2
```

tags are mutable and thus not secure... can you change that to e.g.
```suggestion
  url: https://github.com/SantanderMetGroup/climate4R.climdex/archive/refs/tags/v{{ version }}.tar.gz
  sha256: dce447218dc57730870c68af01291179acf9f4ae67a780cc3bea3a05a9dbbe51
```

Same for all the other packages specified like this.